version-metadata/README.md

```diff
@@ -118,6 +118,63 @@ With step id `version` you'll find the outputs at `steps.version.outputs.OUTPUT_
   run: 'echo "Version has not changed"'
 ```
 
+## Support for repos without `package.json`
```

Comment:
this shouldn't be displayed/advertised so strongly, this is a niche and hacky feature

Reply:
> any project and not just node

I wouldn't call this _niche_